src/v2/components/MessageList/DateSeparator.tsx

```diff
@@ -110,9 +110,33 @@ const DateSeparatorWithContext = <
   );
 };
 
+const areEqual = <
+  At extends UnknownType = DefaultAttachmentType,
+  Ch extends UnknownType = DefaultChannelType,
+  Co extends string = DefaultCommandType,
+  Ev extends UnknownType = DefaultEventType,
+  Me extends UnknownType = DefaultMessageType,
+  Re extends UnknownType = DefaultReactionType,
+  Us extends UnknownType = DefaultUserType
+>(
+  prevProps: DateSeparatorPropsWithContext<At, Ch, Co, Ev, Me, Re, Us>,
+  nextProps: DateSeparatorPropsWithContext<At, Ch, Co, Ev, Me, Re, Us>,
+) => {
+  const {
+    message: { date: prevDate },
+  } = prevProps;
+  const {
+    message: { date: nextDate },
+  } = nextProps;
+
+  const dateEqual = prevDate === nextDate;
```

Comment:
there a reason to do it this way vs `return prevDate === nextDate;` ?

Reply:
nah I messed that up. I think I thought I was gonna use more checks than 1 and forgot to circle back